helpers/handlebars/helpers/index.js

```diff
@@ -33,6 +33,13 @@ module.exports = {
             return opts.inverse(this);
         }
     },
+    ifvalue: (conditional, options) => {
```

Comment:
Isn't `{{#ifvalue this.submission.grade value=11}}` the same as `{{#ifeq this.submission.grade "11"}}` which is already implemented?